salt/metalk8s/kubernetes/cni/calico/deployed.sls

```diff
@@ -4474,15 +4470,8 @@ spec:
             # Disable IPv6 on Kubernetes.
             - name: FELIX_IPV6SUPPORT
               value: "false"
-            # Set Felix logging to "info"
-            - name: FELIX_LOGSEVERITYSCREEN
-              value: "info"
             - name: FELIX_HEALTHENABLED
               value: "true"
-            # Note: We do not want to report about outgoing connections
-            #       in Metalk8s
-            - name: FELIX_USAGEREPORTINGENABLED
-              value: "false"
```

Comment:
I think we still want this to be disabled, MetalK8s should not attempt to send information to the outside.

Reply:
Right, I will add it back